src/osal/lv_freertos.c

```diff
@@ -318,6 +318,52 @@ lv_result_t lv_thread_sync_delete(lv_thread_sync_t * pxCond)
     return LV_RESULT_OK;
 }
 
+lv_result_t lv_thread_sync_signal_isr(lv_thread_sync_t * sync)
```

Comment:
This API need to be tested with USE_FREERTOS_TASK_NOTIFY or without USE_FREERTOS_TASK_NOTIFY.